src/core/metrics.cc

```diff
@@ -310,6 +310,72 @@ instance_id_type shard() {
     return sstring("0");
 }
 
+void
+impl::set_metric_families_to_replicate(
+        std::unordered_map<seastar::sstring, int> metric_families_to_replicate) {
+    // Remove replicas for all metric families that are not
+    // present in the new map.
+    for (const auto& [name, destination]: _metric_families_to_replicate) {
+        if (!metric_families_to_replicate.contains(name)) {
+            remove_metric_replica_family(name, destination);
```

Comment:
`remove_metric_replica_family` isn't introduced until the next commit.

Reply:
Yeah, sorry about that. I moved adding `set_metric_families_to_replicate` to its own commit now.